backend/baton/src/test/java/touch/baton/domain/runnerpost/RunnerPostTest.java

```diff
@@ -276,4 +276,72 @@ void addAllRunnerPostTags() {
         // then
         assertThat(runnerPost.getRunnerPostTags().getRunnerPostTags()).hasSize(2);
     }
+
+    @DisplayName("Supporter 할당")
+    @Nested
+    class AssignSupporter {
+
+        @DisplayName("RunnerPost 내부의 Supporter 가 null 이며 ReviewStatus 가 NOT_STARTED 가 아니어야 하며 Deadline 이 끝나지 않은 경우 성공한다.")
```

Comment:
NOT_STARTED 이어야하며로 수정할게요
```suggestion
        @DisplayName("RunnerPost 내부의 Supporter 가 null 이며 ReviewStatus 가 NOT_STARTED 이어야하며 Deadline 이 끝나지 않은 경우 성공한다.")
```